src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -13179,6 +13846,368 @@ private static Vector512<T> ElementWiseSelect<T>(Vector512<T> mask, Vector512<T>
+                }
+
+                throw new NotSupportedException();
+            }
+        }
+
+        /// <summary>T.PopCount(x)</summary>
+        internal readonly struct PopCountOperator<T> : IUnaryOperator<T> where T : IBinaryInteger<T>
+        {
+            public static bool Vectorizable => false; // TODO: Vectorize
+            public static T Invoke(T x) => T.PopCount(x);
+            public static Vector128<T> Invoke(Vector128<T> x) => throw new NotSupportedException();
+            public static Vector256<T> Invoke(Vector256<T> x) => throw new NotSupportedException();
+            public static Vector512<T> Invoke(Vector512<T> x) => throw new NotSupportedException();
+        }
+
+        /// <summary>T.LeadingZeroCount(x)</summary>
+        internal readonly struct LeadingZeroCountOperator<T> : IUnaryOperator<T> where T : IBinaryInteger<T>
+        {
+            public static bool Vectorizable => false; // TODO: Vectorize
+            public static T Invoke(T x) => T.LeadingZeroCount(x);
+            public static Vector128<T> Invoke(Vector128<T> x) => throw new NotSupportedException();
+            public static Vector256<T> Invoke(Vector256<T> x) => throw new NotSupportedException();
+            public static Vector512<T> Invoke(Vector512<T> x) => throw new NotSupportedException();
+        }
+
+        /// <summary>T.TrailingZeroCount(x)</summary>
+        internal readonly struct TrailingZeroCountOperator<T> : IUnaryOperator<T> where T : IBinaryInteger<T>
+        {
+            public static bool Vectorizable => false; // TODO: Vectorize
+            public static T Invoke(T x) => T.TrailingZeroCount(x);
+            public static Vector128<T> Invoke(Vector128<T> x) => throw new NotSupportedException();
+            public static Vector256<T> Invoke(Vector256<T> x) => throw new NotSupportedException();
+            public static Vector512<T> Invoke(Vector512<T> x) => throw new NotSupportedException();
+        }
+
+        private readonly struct CopySignOperator<T> : IBinaryOperator<T> where T : INumber<T>
+        {
+            // This method is required to work for all inputs, including NaN, so we operate on the raw bits.
+            // Remove the sign from x, and remove everything but the sign from y.
+            // Then OR them to get the correct sign.
+            private const int SingleSignMask = 1 << 31;
+            private const long DoubleSignMask = 1L << 63;
+
+            public static bool Vectorizable => true;
+
+            public static T Invoke(T x, T y) => T.CopySign(x, y);
+
+            public static Vector128<T> Invoke(Vector128<T> x, Vector128<T> y)
+            {
+                if (typeof(T) == typeof(float))
+                {
+                    return
+                        ((x.AsInt32() & Vector128.Create(~SingleSignMask)) |
+                          y.AsInt32() & Vector128.Create(SingleSignMask)).As<int, T>();
```

Comment:
You can make this:
```csharp
return Vector128.ConditionalSelect(Vector128.Create(-0.0f), y, x);
```

Since `ConditionalSelect(cond, left, right) => (left & cond) | (right & ~cond)` and `-0.0f` is equivalent to the sign mask.